app/controllers/api/v1/members_controller.rb

```diff
@@ -3,11 +3,13 @@
 module Api
   module V1
     class MembersController < ApplicationController
+      skip_before_action :authenticate_user!, only: %i[index]
       before_action :set_member, only: %i[show update destroy]
+      after_action { pagy_headers_merge(@pagy) if @pagy }
 
       def index
-        @member = Member.all
-        render json: MemberSerializer.new(@member).serializable_hash
+        @pagy, @member = pagy(Member.all, items: params[:items] || 10, page: params[:page] || 1)
+        render json: MemberSerializer.new(@member, { fields: { member: [:name] } }).serializable_hash
```

Comment:
what is the purpose of this `{ fields: { member: [:name] } }`, maybe you don't need it